jcl/source/windows/JclDebug.pas

```diff
@@ -328,7 +331,7 @@   TJclBinDebugScanner = class(TObject)
     function MakePtr(A: Integer): Pointer;
     class function ReadValue(var P: Pointer; var Value: Integer): Boolean; {$IFDEF SUPPORTS_STATIC}static;{$ENDIF}
   public
-    constructor Create(AStream: TCustomMemoryStream; CacheData: Boolean);
+    constructor Create(AStream: TCustomMemoryStream; CacheData: Boolean; CacheProcNames : boolean);
```

Comment:
Once again, no space before a colon
`Boolean`  should take a capital first letter